docs: update texts in validations-and-constraints.md (#441)
sequelize/website: versioned_docs/version-6.x.x/core-concepts/validations-and-constraints.md

} /* ... */
```

> When this model is [synchronized](./model-basics.md#model-synchronization) (by calling `sequelize.sync`), the `username` field will be created in the table as `` `username` TEXT UNIQUE``. An attempt to insert a username that already exists will throw a `SequelizeUniqueConstraintError`.

Author: ```suggestion
[Synchronize the model](./model-basics.md) by calling `sequelize.sync` to create the `username` field in the table as `` `username` TEXT UNIQUE``. An attempt to insert a username that already exists will throw a `SequelizeUniqueConstraintError`.
```

Here's an alternative.

> When this model is [synchronized](https://sequelize.org/docs/v6/core-concepts/model-basics/#model-synchronization), the `username` field will be created in the table as `` `username` TEXT UNIQUE``. An attempt to insert a username that already exists will throw a `SequelizeUniqueConstraintError`.

Author: ```suggestion
When this model is [synchronized](https://sequelize.org/docs/v6/core-concepts/model-basics/#model-synchronization) (by calling `sequelize.sync`), the `username` field will be created in the table as `` `username` TEXT UNIQUE``. An attempt to insert a username that already exists will throw a `SequelizeUniqueConstraintError`.
```

And, how to link to an internal doc correctly?

Reviewer: You can use a relative link to the markdown file, here is an example: https://github.com/sequelize/website/blob/6fa8e600d7d3cc83dc628857bb6dbb4b1719e2d3/docs/index.mdx?plain=1#L84

});
```

If you need to pass multiple arguments to the built-in validation functions, pass an array. But if a single array argument is to be passed, for example an array of acceptable strings for `isIn`, this is interpreted as multiple string arguments instead of one array argument. To work around this, pass a single-length array of arguments, such as `[['foo', 'bar']]`.

> To use a custom error message instead of built-in validation methods provided by [validator.js](https://github.com/chriso/validator.js), use an object instead of the plain value or array of arguments. For example, include a validator with no argument as a custom message:

Author: ```suggestion
To use a custom error message instead of built-in string validators provided by [validator.js](https://github.com/chriso/validator.js), use an object instead of the plain value or array of arguments. For example, include a validator with no argument as a custom message:

```

}
```

When using custom validator functions, the error message will be whatever message the thrown `Error` object holds.

**Hint:** You can also define a custom function for the logging part. Just pass a function. The first parameter will be the string that is logged.

### `allowNull: true` with custom validators

> If a field that does not allow null values is set to null, all validators will be skipped. A `ValidationError` will be thrown.

Author: ```suggestion
If you null a field that does not allow null values, you will see a `ValidationError`. All validators will be skipped.
```

> If a field that allows null values (`allowNull: true`) is set to null, only the built-in validators will be skipped. Custom validators will still run.

Author: ```suggestion
If you null a field that allows null values (`allowNull: true`), only the built-in validators will be skipped, but custom validators will still run.
```

> This can be useful, for example, if you have a string field that can have a length between 5 and 10 characters but also allows `null`. If the value is `null`, the length validator is skipped:

Author: ```suggestion
This may be useful for string fields that allow 5 to 10 characters but also `null`. If the value is `null`, the length validator is skipped:
```
